### PR TITLE
Adds new biohazard equipment and respirators.

### PR DESCRIPTION
**Pull request by Laniakeaaaa**
* Adds three new biohazard equipment suits!

* Janitorial Suit
* Medical Suit
* Engineering Radiation Suit.

These will spawn in limited quantities in lockers.

*To Do:

- [ ] Finalize locker code for random chance to spawn them.
- [ ] Minor name/wording changes.

*Images:
Showcase

### CODE REVIEW

File: code/modules/clothing/suits/utility.dm [hunk 1]
```diff
@@ -123,4 +123,3 @@
 
 /obj/item/clothing/suit/radiation/New()
 	..()
-	slowdown_per_slot[slot_shoes] = 1.5
```
Comment (Glloyd) [inline code/modules/clothing/suits/utility.dm]: Why was this removed?

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 2]
```diff
@@ -155,4 +155,3 @@
 	siemens_coefficient = 0.7
 	permeability_coefficient = 0.03
 	armor = list(melee = 40, bullet = 40, laser = 40, energy = 25, bomb = 30, bio = 0, rad = 0)
-
```
Comment (Glloyd) [inline code/modules/clothing/gloves/miscellaneous.dm]: Uhh

File: code/modules/clothing/suits/utility.dm (hunk 1 above)
Comment (Laniakeaaaa) [reply code/modules/clothing/suits/utility.dm]: I believe when I originally did it, I started working in that file. I most likely accidently deleted that, will fix.
